internal/sourcemap/elasticsearch.go

```diff
@@ -97,12 +100,33 @@ func (s *esFetcher) Fetch(ctx context.Context, name, version, path string) (*sou
 	if err != nil {
 		return nil, err
 	}
-	return parseSourceMap(body)
+
+	if body == "" {
+		return nil, nil
+	}
+
+	d, err := base64.StdEncoding.DecodeString(body)
+	if err != nil {
+		return nil, fmt.Errorf("failed to base64 decode string: %w", err)
+	}
+
+	r, err := zlib.NewReader(bytes.NewReader(d))
+	if err != nil {
+		return nil, fmt.Errorf("failed to create zlib reader: %w", err)
+	}
+	defer r.Close()
+
+	bbb, err := io.ReadAll(r)
```

Comment:
What is `bbb`? ;)

Reply:
~A result of the grammar { S -> b, S -> Sb} used to name variables~Thanks spotting this! I've updated the variables in this PR so use more meaningful names :+1: